core/src/main/java/nl/nn/adapterframework/senders/JavascriptSender.java

```diff
@@ -207,9 +208,12 @@ public void setJsFunctionName(String jsFunctionName) {
 	}
 
 	/**
-	 * the name of the javascript engine to be used
+	 * the name of the JavaScript engine to be used.
+	 * @deprecated Both Nashorn and Rhino are deprecated. Use J2V8 instead.
 	 * @ff.default J2V8
 	 */
+	@Deprecated
+	@ConfigurationWarning("JavaScript engines Nashorn and Rhino deprecated. Use \"J2V8\" instead")
```

Comment:
Please also add here that Nashorn and Rhine will be deleted in 8.0

Reply:
As discussed in standup, not needed.

Reply:
Thanks